roles/migrationcontroller/tasks/main.yml

```diff
@@ -155,74 +175,78 @@
           state: "{{ velero_state }}"
           definition: "{{ lookup('template', 'velero_supporting.yml.j2') }}"
 
-  - name: "Create proxy configuration secret if proxy is configured"
-    k8s:
-      state: "present"
-      definition: "{{ lookup('template', 'proxy_secret.yml.j2') }}"
-    when: http_proxy != "" or https_proxy != "" or no_proxy != ""
+  - when: reconcile_migrationcontroller
```

Comment:
This makes the tenant consumption of the proxy interesting... since we are dependent on the existence of a migrationcontroller to install a tenant, it would follow that the tenants should be consuming this secret if it exists.

Reply:
@dymurray agreed